File: SocialPulse/SocialPulse.Core/Entities/Question.cs
namespace SocialPulse.Core
{
    public class Question : BaseEntity
    {
        public string Text { get; set; } = null!;
        public int UserId { get; set; }
        public User User { get; set; } = null!;
        public Answer? Answer { get; set; }
    }
}

File: SocialPulse/UI/mobile_socialpulse/lib/models/question.dart
import 'package:json_annotation/json_annotation.dart';

import 'answer.dart';

part 'question.g.dart';

@JsonSerializable()
class Question{
  int? id;
  String? text;
  int? userId;
  Answer? answer;

  Question (this.id,this.text,this.userId, this.answer);

  factory Question.fromJson(Map<String, dynamic> json) =>
      _$QuestionFromJson(json);

  Map<String, dynamic> toJson() => _$QuestionToJson(this);
}